### PR TITLE
Newsletter: improve paywall on emails

### LINKED ISSUE #32334: Paywall email: Update design
When we send an email with a post that has a paywall, we need to add a new module that nudges free subscribers to upgrade their subscription to a paid one.

## Design
The goal is to make a visually distinctive section that's different from the post itself.

image

[Figma link](qQxuiiWjgNV0AiYg6070eo-fi-82_13796).

### PR DESCRIPTION
**Pull request by lezama**
Fixes #32334

Screenshot 2023-08-08 at 11 08 11

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to '..'
*

### CODE REVIEW

**Comment by simison** [comment]
Rebased for easier testing.

File: projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php [hunk 1]
```diff
@@ -846,3 +850,56 @@ function get_paywall_blocks( $newsletter_access_level ) {
...
+        <!-- /wp:paragraph -->
+
+        <!-- wp:buttons -->
+        <div class="wp-block-buttons" style="text-align: center;">
+            <!-- wp:button -->
+            <div class="wp-block-button" style="display: inline-block; margin: 10px 0;">
+                <a href="#" class="wp-block-button__link wp-element-button"
+                   style="display: inline-block;
+                          padding: 15px 20px;
+                          background-color: #0675C4;
```
Comment (simison) [inline projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php:888]: For Lettre theme we have some code that changes the links and buttons to the theme's primary colour.

I think there's a helper function somewhere that does that... Try change Lettre style from Site editor (you need a premium plan) and then send a test email.

Apex implemented it so they'd know more.

Comment (lezama) [reply projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php:888]: The plan is to use the subscribe block once it renders on emails. The subscribe block should already do this for us.

**Review by simison — APPROVED** [review]
Tests well:

Site:
image

Email (Apple mail app):

image

Email (Gmail in Firefox):

image

Logged in:

image

**Comment by simison** [comment]
Rebased

**Pull request merged**